README.rst

```diff
@@ -93,7 +93,75 @@ If your network connection is lost, you will get below error message.
     >>> ndic.search('...')
     NdicConnectionError: Network connection is lost. Please check the connection to the Internet.
 
-Command Line Interface
+Chinese Usage
+-----
+
+The usage is very similar with English Usage.
+
+Begin by importing the Ndic module:
+
+.. code-block:: python
+
+    >>> import ndic
+
+Entering an English word as the ``search`` function argument will return the
+corresponding Korean word(s).
+
+.. code-block:: python
+
+    >>> ndic.search_zh('苹果')
+    [
+      {
+         'entryNameTTS': '苹果',
+         'meanList': [
+            {
+               'meaning': '사과(나무).',
```

Comment:
Sorry for late review 😭. I was a little busy because of my company work.

> `[ { 'origin': 'string', 'meaning': 'string', 'pinyi': 'string', }, { 'origin': 'string', 'meaning': 'string', 'pinyi': 'string', }, ]`

This format seems better than before.
Actually, I don't know Chinese at all, so I don't know well which field users want to get.
If you think these 3 fields(`origin`, `meaning`, `pinyi`) are enough, I think it is much simpler and more readable.

Reply:
@jupiny i guess i finished my work would you review in your spare time?

Reply:
Thank you for hard work! Let me review it later 😄